packages/build-utils/src/debug.ts

```diff
@@ -1,9 +1,6 @@
-import { getPlatformEnv } from './get-platform-env';
-
-export default function debug(message: string, ...additional: any[]) {
-  if (getPlatformEnv('BUILDER_DEBUG')) {
-    console.log(message, ...additional);
-  } else if (process.env.VERCEL_DEBUG_PREFIX) {
-    console.log(`${process.env.VERCEL_DEBUG_PREFIX}${message}`, ...additional);
+export default function debug(message: string) {
+  if (process.env.VERCEL_DEBUG_PREFIX) {
+    const line = message.replace(/\r?\n/g, ' ');
```

Comment:
I think this is a bit too restrictive. I would recommend continuing to accept `message` and `additional`, but pass them into `util.format()`: https://nodejs.org/dist/latest-v18.x/docs/api/util.html#utilformatformat-args. Then split on lines and call `console.log()` for each line:

```
const lines = util.format(message, ...additional).split(/\r?\n/);
for (const line of lines) {
  console.log(line);
}
```